presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergAbstractMetadata.java

```diff
@@ -450,17 +554,4 @@ private long removeScanFiles(Table icebergTable, Iterable<FileScanTask> scan)
         transaction.commitTransaction();
         return rowsDeleted.get();
     }
-
-    @Override
-    public void truncateTable(ConnectorSession session, ConnectorTableHandle tableHandle)
-    {
-        IcebergTableHandle handle = (IcebergTableHandle) tableHandle;
-        Table icebergTable = getIcebergTable(session, handle.getSchemaTableName());
-        try (CloseableIterable<FileScanTask> files = icebergTable.newScan().planFiles()) {
-            removeScanFiles(icebergTable, files);
-        }
-        catch (IOException e) {
-            throw new PrestoException(GENERIC_INTERNAL_ERROR, "failed to scan files for delete", e);
-        }
-    }
```

Comment:
I noticed that in the new changes, you still have a `truncateTable` function. Are there any differences between the deleted and the new `truncateTable`?

Reply:
They are exactly the same. The purpose of moving `truncateTable` function up is just to leave the private function `removeScanFiles` at the bottom.